packages/skill-lesson/admin/coupon-generator-form.tsx

```diff
@@ -72,13 +72,15 @@ const CouponGeneratorForm = () => {
 
   return (
     <Form {...form}>
-      <form onSubmit={form.handleSubmit(onSubmit)} className="space-y-8">
-        <fieldset className="grid-cols-4 gap-5 md:grid">
+      <form onSubmit={form.handleSubmit(onSubmit)}>
+        <fieldset className="grid-cols-4 gap-5 md:grid space-y-5 md:space-y-0">
```

Comment:
@vojtaholik for some (mystic!) reason `md:grid-cols-4` didn't work for me so I used `space-y` class instead of `gap-`.